### ISSUE TITLE
Available on PCBway?

### ISSUE THREAD

**Issue by csembroski** [Gavin-Watkins/OCXO_Clock#1]
Love the look of this project.  Have you shared the PCB setup at PCBway?  I'm great with a solder iron, but I have no experience getting boards manufactured.  I believe PCBway has a way to publish the setup on their platform?

**Comment by Gavin-Watkins** [comment]
Hi,
Thanks for your interest in my project. I've not shared the PCB on PCBway. I probably can do that. Have you seen the details on my GitHub?
ThanksGavin

Yahoo Mail: Search, organise, conquer 
 
  On Sat, 30 Mar 2024 at 14:14, Chuck ***@***.***> wrote:   


Love the look of this project. Have you shared the PCB setup at PCBway? I'm great with a solder iron, but I have no experience getting boards manufactured. I believe PCBway has a way to publish the setup on their platform?

**Comment by csembroski** [comment]
Yes, I did look at your github but I’ve never had boards manufactured before.  I’m nearing retirement and want to build some projects before my eyes give out.  lolI built one of those Russian Nixie tube clocks a couple years ago but my wife got tangled up in the cord and it broke when it hit the tile floor.  Sent from my iPhoneOn Mar 30, 2024, at 1:19 PM, Gavin Watkins ***@***.***> wrote:﻿
Hi,

Thanks for your interest in my project. I've not shared the PCB on PCBway. I probably can do that. Have you seen the details on my GitHub?

ThanksGavin



Yahoo Mail: Search, organise, conquer 

 

  On Sat, 30 Mar 2024 at 14:14, Chuck ***@***.***> wrote:   





Love the look of this project. Have you shared the PCB setup at PCBway? I'm great with a solder iron, but I have no experience getting boards manufactured. I believe PCBway has a way to publish the setup on their platform?

**Comment by Gavin-Watkins** [comment]
Sorry to hear about your nixie tube clock. I've never built one, but have quite a lot of valve/tube radios and hifi equipment around the house that I'm pretty paranoid about.
I used JLC PCB, who were fairly straightforward. You just need to upload the gerbers in a zip file. They extract them and fabricate the PCB. A couple of guys at the IEEE Spectrum also made the clock to verify my design. I'm in the process of moving home - and actually country too. I may still have a few spare PCBs. Generally when you order it is quantites of five or so. All my stuff is being picked up tomorrow and it will be two months until we're able to start unpacking.
 
ThanksGavin

    On Monday, 1 April 2024 at 02:30:15 BST, Chuck Sembroski ***@***.***> wrote:  
 
 

Yes, I did look at your github but I’ve never had boards manufactured before.  I’m nearing retirement and want to build some projects before my eyes give out.  lolI built one of those Russian Nixie tube clocks a couple years ago but my wife got tangled up in the cord and it broke when it hit the tile floor.  Sent from my iPhoneOn Mar 30, 2024, at 1:19 PM, Gavin Watkins ***@***.***> wrote:﻿
Hi,

Thanks for your interest in my project. I've not shared the PCB on PCBway. I probably can do that. Have you seen the details on my GitHub?

ThanksGavin



Yahoo Mail: Search, organise, conquer 

 

 On Sat, 30 Mar 2024 at 14:14, Chuck ***@***.***> wrote: 





Love the look of this project. Have you shared the PCB setup at PCBway? I'm great with a solder iron, but I have no experience getting boards manufactured. I believe PCBway has a way to publish the setup on their platform?

**Comment by csembroski** [comment]
Understood.  I built a tube amp from a kit a while ago, but I haven’t gotten much time to listen.  I took a programming job at NASA JSC working on the Artemis Gateway moon project and it sucks up all my free time just getting my feet under me.

I’ll put it on my todo list to get your gerbers and try JLC PCB.  

Best wishes on a smooth move,
Chuck
